src/ui/Network/Network.tsx

```diff
@@ -0,0 +1,41 @@
+import { loginSupportedChainIds } from 'common-lib/constants';
```

Comment:
i would put this in src/features/auth since it's specific to our app

Reply:
src/features/auth/NetworkSelector/, even

Reply:
done.